frontend/src/containers/RestaurantListPage.jsx

```diff
@@ -1,7 +1,14 @@
 import React from "react";
+import TagsTab from '../components/RestaurantList/TagsTab';
+import "../components/RestaurantList/style.css";
 
-const RestautantListPage = () => {
-  return <div>Rest list</div>;
+const RestautantListPage = (props) => {
+  return (
+      <div className="container-main">
```

Comment:
Use <React.Fragment> here instead <div className="container-main"> cuz we have a PageContainer component that render our Layout